plugins/rsyslog.yaml

```diff
@@ -56,6 +92,10 @@ parameters:
 # {{$connection_type := default "udp" .connection_type}}
 # {{$protocol := default "rfc5424 (IETF)" .protocol}}
 # {{$location := default "UTC" .location}}
+# {{$tls_enable := default false .tls_enable}}
+# {{$tls_certificate := default "" .tls_certificate}}
+# {{$tls_private_key := default "" .tls_private_key}}
+# {{$tls_min_version := default "" .tls_min_version}}
```

Comment:
This should be set even if it is not used by default, matching the default value for the parameter. I used 1.2 but Go can go down to 1.1 and uses 1.1 as the default minimum. Use whatever you see fit. I am a fan of starting high and allowing the user to go lower.
```suggestion
# {{$tls_min_version := default "1.2" .tls_min_version}}
```